server/consumer.go

```diff
@@ -69,6 +70,11 @@ type ConsumerConfig struct {
 	Direct bool `json:"direct,omitempty"`
 }
 
+type PushActive struct {
+	Subject string
+	Queue   string
```

Comment:
Should this have json tag and specifically Queue be omitempty?

Reply:
lol yes. One sec.

Reply:
ok pushed.